service/src/test/java/com/commercetools/pspadapter/payone/domain/payone/PayonePostServiceImplTest.java

```diff
@@ -92,29 +94,29 @@ public void getObjectMapWithExpandedLists() {
-        assertThat(withExpandedLists.get("empty")).isEqualTo("");
-        assertThat(withExpandedLists.get("boolTrue")).isEqualTo(true);
-        assertThat(withExpandedLists.get("boolFalse")).isEqualTo(false);
-        assertThat(withExpandedLists.get("list1[1]")).isEqualTo(1);
-        assertThat(withExpandedLists.get("list1[2]")).isEqualTo(2);
-        assertThat(withExpandedLists.get("list1[3]")).isEqualTo(3);
-        assertThat(withExpandedLists.get("listString[1]")).isEqualTo("ein");
-        assertThat(withExpandedLists.get("listString[2]")).isEqualTo("zwei");
-        assertThat(withExpandedLists.get("listString[3]")).isEqualTo("drei");
-        assertThat(withExpandedLists.get("listDoubles[1]")).isEqualTo(3.14);
-        assertThat(withExpandedLists.get("listDoubles[2]")).isEqualTo(2.71);
-        assertThat(withExpandedLists.get("listDoubles[3]")).isEqualTo(9.81);
-
-
-        final Map<String, Object> withEmptyLists = payonePostService.getObjectMapWithExpandedLists(
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("foo", "bar"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("woot", "wootValue"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("a", "42"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("empty", ""));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("boolTrue", "true"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("boolFalse", "false"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("list1[1]", "1"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("list1[2]", "2"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("list1[3]", "3"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("listString[1]", "ein"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("listString[2]", "zwei"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("listString[3]", "drei"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("listDoubles[1]", "3.14"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("listDoubles[2]", "2.71"));
+        assertThat(withExpandedLists).contains(new BasicNameValuePair("listDoubles[3]", "9.81"));
+
+
+        final List<BasicNameValuePair> withEmptyLists = payonePostService.getNameValuePairsWithExpandedLists(
                 ImmutableMap.of("foo", new ArrayList<>(),
                         "bar", new LinkedList<>()));
 
         assertThat(withEmptyLists.size()).isEqualTo(2);
-        assertThat(withEmptyLists.get("foo[]")).isEqualTo("");
-        assertThat(withEmptyLists.get("bar[]")).isEqualTo("");
+        assertThat(withEmptyLists).contains(new BasicNameValuePair("foo[]", ""));
+        assertThat(withEmptyLists).contains(new BasicNameValuePair("bar[]", ""));
```

Comment:
also here you could use 
````java
containsExactly(new BasicNameValuePair("foo[]", ""), new BasicNameValuePair("bar[]", ""))
````

Reply:
nice catch, we had just merged newest assertJ last wee, so we will re-factor step-by-step such tests in the future
fbbabb8d